packages/pwa/app/commerce-api/auth.js

```diff
@@ -165,18 +213,18 @@ class Auth {
         if (this._pendingLogin) {
             return this._pendingLogin
         }
-
         let retries = 0
         const startLoginFlow = () => {
             let authorizationMethod = '_loginAsGuest'
             if (credentials) {
                 authorizationMethod = '_loginWithCredentials'
-            } else if (this._refreshToken) {
+            } else if (this.refreshToken) {
                 authorizationMethod = '_refreshAccessToken'
             }
             return this[authorizationMethod](credentials)
                 .catch((error) => {
-                    if (retries === 0 && error.message === 'EXPIRED_TOKEN') {
+                    const retryErrors = [INVALID_TOKEN, EXPIRED_TOKEN]
+                    if (retries === 0 && retryErrors.includes(error.message)) {
```

Comment:
The array here is a constant, it could also be extracted to the top of the file. Or you could do `===` checks instead of `[].includes`.